src/ln/channel.rs

```diff
@@ -679,8 +680,10 @@ impl Channel {
 		let txins = {
 			let mut ins: Vec<TxIn> = Vec::new();
 			ins.push(TxIn {
-				prev_hash: self.channel_monitor.get_funding_txo().unwrap().txid,
-				prev_index: self.channel_monitor.get_funding_txo().unwrap().index as u32,
+				previous_output: BitcoinOutPoint {
```

Comment:
Implementing `From<BitcoinOutPoint> for OutPoint` can make this more cleaner, you just call `self.channel_monitor.get_funding_txo().unwrap().into()`.